docs/docker-dev/docker-dev-setup-manual.md

```diff
@@ -0,0 +1,18 @@
+# Docker Development Environment Setup
+
```

Comment:
I think we need an introductory paragraph that explains why a user might choose this setup, and what following these steps will actually do, at a high level.

Reply:
Agree, will add one.